pxr/imaging/plugin/hdRpr/camera.cpp

```diff
@@ -113,7 +113,7 @@ void HdRprCamera::Sync(HdSceneDelegate* sceneDelegate,
         EvalCameraParam(&m_apertureBlades, RprUsdTokens->rprCameraBlades, sceneDelegate, id, 16u);
     }
 
-    if (*dirtyBits & HdCamera::DirtyViewMatrix) {
+    if (*dirtyBits & HdCamera::DirtyTransform) {
```

Comment:
Good observation - this enum entry is aliased since USD v21.02, so I changed the version respectively:
https://github.com/PixarAnimationStudios/USD/blob/v21.02/pxr/imaging/hd/camera.h#L100